src/PixiEditor/Models/Rendering/SceneRenderer.cs

```diff
@@ -265,61 +360,36 @@ private bool HighDpiRenderNodePresent(IReadOnlyNodeGraph documentNodeGraph)
 
         return highDpiRenderNodePresent;
     }
+}
 
-    private void RenderOnionSkin(DrawingSurface target, ChunkResolution resolution, SamplingOptions sampling, string? targetOutput)
+struct RenderState
```

Comment:
Not quite as important as the struct isn't passed around much, but could be made a `readonly` struct (pretty huge at 70 bytes), you can use init; instead of set; for the properties